Student-Unity/src/main/java/csed/swe/studentunity/model/User.java

```diff
@@ -19,7 +19,8 @@ public class User implements Serializable {
 
     @Id
     @GeneratedValue(strategy = GenerationType.SEQUENCE)
-    @Column(name = "id", nullable = false, columnDefinition = "INTEGER")
+    @Column(name = "id", updatable = false)
```

Comment:
why did you remove ``columnDefinition = "INTEGER"``

Reply:
Actually i didn't remove it for intended reason someone else have added it and i didn't make a pull before i pushed my changes so it was removed.
I have no problem to keep it.